en/docs/fragments/download-saml-metadata.md

```diff
@@ -0,0 +1,7 @@
+1.  Login to WSO2 IS as the administrator of the tenant. 
```

Comment:
We need to replace the fragment from the portal docs right? Can we add that standard TODO here?